app/build.gradle

```
@@ -201,6 +201,8 @@ dependencies {
 
     // Preferences
     implementation 'com.github.tfcporciuncula:flow-preferences:1.3.0'
+    implementation 'com.github.ByteHamster:SearchPreference:v1.0.3'
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
```

```suggestion
    implementation "org.jetbrains.kotlin:kotlin-reflect:$BuildPluginsVersion.KOTLIN"
```

Also maybe put it where the build.gradle contains the kotlin stdlib

---

I'll fix this.